docs/managing-teams.md

```diff
@@ -21,18 +21,18 @@ organisation:
 
 programmes: [] # A list of programmes (flu, hpv, menacwy, td_ipv)
 
-teams:
-  team1: # Identifier to link team with school and links below, not used in app
+subteams:
+  subteam1: # Identifier to link team with school and links below, not used in app
```

Comment:
Small nit; we're using `team` and `subteam` here and it's not 100% clear whether that's the same concept or different ones

Reply:
oh right, having seen https://github.com/nhsuk/manage-vaccinations-in-schools/pull/4038, I'm with you now.